Doppler.BillingUser/Controllers/BillingController.cs

```diff
@@ -0,0 +1,39 @@
+using Doppler.BillingUser.DopplerSecurity;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.BillingUser.Controllers
+{
+    [Authorize]
+    [ApiController]
+    public class BillingController
+    {
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/currentbillinginformation")]
+        public string GetCurrentBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to GetCurrentBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpPut("/accounts/{accountname}/billinginformation")]
+        public string UpdateBillingInformation(string accountname)
+        {
+            return $"Hello! \"you\" that have access to UpdateBillingInformation with accountname '{accountname}'";
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        [HttpGet("/accounts/{accountname}/current-payment-method")]
```

Comment:
I do not remember what we said about this, but probably we decided that each payment method will be unique, there is no update, for that reason we decided to use the word _current_, in that case, what do you think about this URL?

```suggestion
        [HttpGet("/accounts/{accountname}/payment-methods/current")]
```

Reply:
Yes, maybe it's better because it will prepared for the future, in case of it's neccesary get all payment methotds for exaple. Thanks!